src/components/common/DocumentDelivery.vue

```diff
@@ -58,6 +67,13 @@ export default class DocumentDelivery extends Vue {
   @Prop({ default: false }) readonly editableCompletingParty: boolean
   @Prop({ default: false }) readonly showCustodianEmail: boolean
   @Prop({ default: false }) readonly invalidSection: boolean
+  // additionaLabel can be used for additon fields.
+  //  For SP can be used for proprietor
+  // FOr GP label can be Partners
+  @Prop({ default: null }) readonly additionalLabel: string
+  // additionalEmail  For SP can be used for single email
+  // For GP additionalEmail can be multipel with comma separated
+  @Prop({ default: null }) readonly additionalEmail: string
```

Comment:
do we need to update ```additionalValue``` thoughts?

Reply:
What is `additionalValue`? Not used?

All existing rows have a label and value. This is similar so I'm OK with it.

Reply:
So i will leave as it is now :)

Reply:
I changed my mind! I think I prefer `additionalValue` (to be consistent with contactLabel and contactValue).